buildSrc/version.properties

```diff
@@ -1,5 +1,4 @@
-# When updating elasticsearch, please update 'rest' version in core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
```

Comment:
I assume this is outdated, the file didn't contain a rest version and the history indicated the need for it has been removed